tests/iscsi/iscsi_client.pm

```diff
@@ -87,8 +89,10 @@ sub initiator_discovered_targets_tab {
 sub initiator_connected_targets_tab {
     # go to discovered targets tab
     send_key "alt-d";
-    record_soft_failure('bsc#1191112 - Resizing window as workaround for YaST content not loading');
-    send_key_until_needlematch('iscsi-initiator-discovered-targets', 'alt-f10', 10, 2);
+    if (is_sle('=15-SP4')) {
+        record_soft_failure('bsc#1191112 - Resizing window as workaround for YaST content not loading');
+        send_key_until_needlematch('iscsi-initiator-discovered-targets', 'alt-f10', 10, 2);
+    }
```

Comment:
same here, we might need an else with a needle check for the `'iscsi-initiator-discovered-targets'` tag.

Reply:
Updated, thanks.